packages/frameworks/package.json

```diff
@@ -0,0 +1,6 @@
+{
+  "name": "@now/frameworks",
+  "version": "0.0.1",
+  "main": "frameworks.json",
```

Comment:
This seems really similar to the framework detection that was added to `@now/build-utils` by @TooTallNate and @AndyBitz except this is just static files? Why?

Reply:
@styfle we're gonna use this as a standalone lib, which will be used by both API, docs and front. cc @leo